tests/unit/src/kakarot/precompiles/test_precompiles.py

```diff
@@ -0,0 +1,31 @@
+import random
+import re
+
+import pytest
+import pytest_asyncio
+from starkware.starknet.testing.contract import StarknetContract
+from starkware.starknet.testing.starknet import Starknet
+
+
+@pytest_asyncio.fixture(scope="module")
+async def precompiles(starknet: Starknet):
+    return await starknet.deploy(
+        source="./tests/unit/src/kakarot/precompiles/test_precompiles.cairo",
+        cairo_path=["src"],
+        disable_hint_validation=True,
+    )
+
+
+@pytest.mark.asyncio
+class TestPrecompiles:
```

Comment:
here you create one test function per cairo test function. You can also create sub classes for grouping tests of the same function, like

```
class TestPrecompiles:
    class TestIsPrecompile:
        async def test_should_return_false_when_address_is_greater_than_last_precompile():
            await precompile.func test_run_should_return_an_execution_context_with_proper_calling_context()
...
```